### ISSUE TITLE
Instead of our current Profiles and Extensions section, just have a Registry section?

### ISSUE THREAD

**Issue by xmlgrrl** [KantaraInitiative/wg-uma#307]
Is it viable/sensible to create an UMA Profiles and Extensions Registry section, with a Registry Template subsection, instead of the current section (latest version in [Sec 4](https://docs.kantarainitiative.org/uma/ed/oauth-uma-ticket-2.0-00.html#profiles) of Grant)? Our original section was based on SAML's material from a long time ago, and that's what has pretty much eventually morphed into the current IANA registries. Can we do a real IANA registry or at least an IANA-like registry?

Core has this section now. Both Grant and FedAuthz would need an equivalent section.

**Comment by xmlgrrl** [comment]
In [UMA telecon 207-04-27](http://kantarainitiative.org/confluence/display/uma/UMA+telecon+2017-04-27), we discussed the relatively rare circumstances of establishing an IANA registry; they certainly don't include us.

Does it make sense, as an alternative, to recommend one short template each for recording details of a) claim token formats, b) UMA security profiles, and c) UMA profiles and profile/extensions for use with APIs? There are existence proofs of b and c (and a, sort of). All of these would be candidates for listing in `uma_profiles_supported`. And we could say anyone could make up any other template they need.

Note: When it came down to it, I ended up just pointing to the Grant version of the section every time I needed to talk about profiles and extensions, so it wouldn't need to appear in both specs; FedAuthz really does just layer on Grant.
